### PR TITLE
Fixing types and case issue in Plaid link

### PR DESCRIPTION
**Pull request by vaughansharman**
Your code was helpful, many thanks

In understanding it I found a few problems and wanted to give back so others doesn't encounter the same problem

Problems encountered
- some types were `String` instead of `string`
- There was an unused type and enum
- The plaid url was named `view` instead of `View` which was causing it to break

Cheers

### CODE REVIEW

File: PlaidLink.tsx [hunk 1]
```diff
@@ -104,7 +104,7 @@ export default function PlaidLink({
   return (
     <WebView
       source={{
-        uri: `https://cdn.plaid.com/link/v2/stable/link.html?isWebview=true&token=${linkToken}`,
+        uri: `https://cdn.plaid.com/link/v2/stable/link.html?isWebView=true&token=${linkToken}`,
```
Comment (JBaczuk) [inline PlaidLink.tsx:107]: Why was this changed? It looks like it should be `isWebview` according to the docs:

https://plaid.com/docs/link/webview/#create

**Review by JBaczuk — COMMENTED** [review]
Thanks for your help, really, I appreciate it a lot, one question...

File: PlaidLink.tsx (hunk 1 above)
Comment (vaughansharman) [reply PlaidLink.tsx:107]: Good call, I thought this was part of the problem and was sure that it wasn't loading without the capital `V` but I have re-tested just now and it's working fine for me. I'll revert this part of the PR

**Review by JBaczuk — APPROVED** [review]
